compose/build.gradle.kts

```diff
@@ -94,9 +94,20 @@ subprojects {
         // IntelliJ refuses to sort imports correctly.
         // This is a known issue: https://github.com/pinterest/ktlint/issues/527
         "import-ordering",
```

Comment:
A heads-up just in case you're interested in a follow-on PR: it looks like the import-ordering issue [has been (mostly) fixed](https://youtrack.jetbrains.com/issue/KTIJ-4813#focus=Comments-27-4205728.0-0) thanks to the introduction of a [layout table for Kotlin imports](https://github.com/JetBrains/kotlin/pull/3336). (But note the misbehavior cited at the end of that conversation, including a comment by the tried-and-true Valeriy Ovechkin.)

Reply:
I'm not interesting in diverging these effectively duplicate configs. We can follow up and address in the main project once these have merged.